src/pperm.c

```diff
@@ -2539,7 +2471,7 @@ Obj FuncShortLexLeqPartialPerm(Obj self, Obj f, Obj g)
         if (DEG_PPERM4(g) == 0)
             return False;
         rankg = RANK_PPERM4(g);
-        domg = DOM_PPERM(f);
+        domg = DOM_PPERM(g);
```

Comment:
Oops, good catch!

Reply:
I couldn't get the code to enter one case, which lead me to spot this 👍